### PR TITLE
Update tvm-software-inventory.md

### PR DESCRIPTION
**Pull request by msbemba**
Added more information about data freshness

Reference - https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/how-is-the-software-inventory-created-in-mdatp/m-p/1447930?lightbox-message-images-2543838=295550i8196483D27428F00

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23725

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by siosulli** [comment]
Thanks @msbemba - I am just double checking this accurate before I publish! 

Thanks,
Sinead

File: microsoft-365/security/defender-vulnerability-management/tvm-software-inventory.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ Access the software inventory page by selecting **Software inventory** from the
 
 The **Software inventory** page opens with a list of software installed in your network, including the vendor name, weaknesses found, threats associated with them, exposed devices, impact to exposure score, and tags.
 
+The data freshness is currently 4 hours. There is currently no way to force a sync.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-vulnerability-management/tvm-software-inventory.md]:
```suggestion
The data is updated every three to four hours. There is currently no way to force a sync.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by siosulli** [comment]
> @siosulli Please merge this PR. Thanks

Will do! I'm still confirming whether this is accurate.

**Comment by yogkumgit** [comment]
@siosulli

**Comment by yogkumgit** [comment]
@siosulli Kindly update. Thanks

**Pull request merged**